### PR TITLE
Gor 885 - Export mag-popover module to gorila-ui-components

### PR DESCRIPTION
**Pull request by tigredonorte**

### CODE REVIEW

File: libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.no-margin {
+  margin: 0 !important;
```
Comment (DenisOkawa) [inline libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss]: The `!important` doesn't seems necessary as it's only applyed to span tags

Comment (tigredonorte) [reply libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss]: In that case it is, because in another place the span margins are setted.

Comment (guilhermejcgois) [reply libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss]: who sets this margin?

Comment (tigredonorte) [reply libs/shell/popups-modals/src/lib/ui/popover/popover.component.scss]: widgets-system/*/style.scss

pre span {
  &:not(:first-child) {
    margin-top: -18px;
  }
}

**Comment by tigredonorte** [comment]
> Missing documentation for new component

done

**Comment by guilhermejcgois** [comment]
close #53

**Pull request merged**